monitor/internal/pod/monitor.go

```diff
@@ -183,8 +183,8 @@ func (m *PodMonitor) Run(ctx context.Context) error {
 	case err := <-errCh:
 		return fmt.Errorf("pod: %s", err.Error())
 	case <-time.After(5 * time.Second):
-		// we give the controller 5 seconds to report back
-		return errors.New("pod: controller did not start within 5s")
+		// we give the controller 5 seconds to report back before we issue a warning
```

Comment:
Also, if we just return from here, there will be a mis-leading log saying enforcerd is ready, where as we are still retrying.  So, we can make the log more verbose.

Reply:
yes, I agree